yb-voyager/src/migassessment/sizing.go

```diff
@@ -95,91 +169,81 @@ Returns:
 
 	[]SourceDBMetadata: A slice containing metadata for sharded objects. Used for sharding recommendations.
 	float64: The total size of sharded objects in gigabytes. Used for sharding recommendations.
+	ExpDataColocatedLimit: Information about the instance type to use for the experiment
+	SourceDBMetadata[]: list of sharded objects
```

Comment:
We have a standard practice of using const varaibles for all tables names.
Let do the same for `colocated_limits`

Reply:
same for `sharded_sizing` table name below

Reply:
done